docs/DeveloperGuide.md

```diff
@@ -315,13 +315,17 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
 2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+4. The code should be open source.
+5. Should not require internet connection.
```

Comment:
Something just came to mind, as we implement v1.2 we can add all the NFRs related to our Mailing feature too:)

Reply:
Sure will add them later then